src/styles/page/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat center;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
```

Comment:
It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars